app/values/scanned_resource_pdf/canvas.rb

```diff
@@ -1,5 +1,7 @@
 class ScannedResourcePDF
   class Canvas
+    LETTER_WIDTH = 612
```

Comment:
OK, now I see where 612 and 792 are coming from -- number of points in a 8.5x11 page.  Perhaps I should have expected the units to be points (because PostScript), but I was expecting pixels instead.  Maybe a comment here or at canvas_downloader#scale_factor making the units explicit would be a good idea?